sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java

```diff
@@ -693,22 +702,25 @@ public void deleteAllSelectedItems(Collection<? extends ContactIndexDto> selecte
 				Type.WARNING_MESSAGE,
 				false).show(Page.getCurrent());
 		} else {
-			VaadinUiUtil.showDeleteConfirmationWindow(
+			DeletableUtils.showDeleteWithReasonPopUp(
 				String.format(I18nProperties.getString(Strings.confirmationDeleteContacts), selectedRows.size()),
-				new Runnable() {
+				(deleteDetails) -> {
+//							new Runnable() {
```

Comment:
Remove commented code if it's no longer needed

Reply:
removed